src/odb/include/odb/db.h

```diff
@@ -7634,7 +7634,7 @@ class dbScanChain : public dbObject
 class dbScanInst : public dbObject
 {
  public:
-  enum SCAN_INST_TYPE
+  enum class SCAN_INST_TYPE
```

Comment:
Fwiw I find dbScanInst::SCAN_INST_TYPE::.. rather redundant.

Reply:
Totally agree, I will address this in a future PR